modules/apps/frontend-theme/kyoung-liferay-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key=" lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+    <title>${the_title} - ${company_name}</title>
+
+    <meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+    <#--  pt animation js for header -->
+    <script src="/o/kyoung-liferay-theme/js/pt.min.js"></script>
+
+    <@liferay_util["include"] page=top_head_include />
+
+</head>
+
+<body class="${css_class}">
+
+    <@liferay_ui["quick-access"] contentId="#main-content" />
+
+    <@liferay_util["include"] page=body_top_include />
+
+    <@liferay.control_menu />
+
+    <#-- Start Header -->
+    <#if show_header>
+        <header role="banner">
+            <div class="flex height-fix" id="home">
+                <div class="canvas" id="pt" ></div>
+				<#-- Start Logo -->
+                <div class="flex">
+                    <div class="mu-logo-area">
+                        <a class="mu-logo" href="#" alt="logo"><img class="logo" alt="img logo ${logo_description}" src="${site_logo}"></a>
+                    </div>
+                    <#assign preferences=freeMarkerPortletPreferences.getPreferences({"portletSetupPortletDecoratorId":
+                        "barebone" , "destination" : "/search" }) />
+                </div>
+                <#-- End Logo -->
+            </div>
+        </header>
+    <#-- End Header Content -->
+
+        <#-- Start Menu -->
+        <button class="mu-menu-btn">
+        </button>
+        <div class="mu-menu-full-overlay">
+            <div class="mu-menu-full-overlay-inner">
+				<a class="mu-menu-close-btn"></a>
+                <nav class="mu-menu" role="navigation">
+                    <ul>
+						<li>
+							<#if show_site_name>
+								<div class="company-name">Welcome To ${site_name}</div>
+							</#if>
+						</li>
```

Comment:
We should wrap the HTML in the Freemarker to not display an empty `<li />` if the element does not appear.
Ex.
```
<#if show_site_name>
	<li>
		<div class="company-name">Welcome To ${site_name}</div>
	</li>
</#if>
```